docs/contributing.rst

```diff
@@ -7,40 +7,33 @@ Contributing
 Hacking
 =======
 
-All changes in your pull request **must** have 100% unit test coverage, pass
-our `integration`_ tests, **and** be compliant with the
-:ref:`coding style <coding-style>`.
+Running a local copy of the client
+----------------------------------
 
-
-Bootstrap
----------
-
-Start by :ref:`installing Let's Encrypt prerequisites
-<prerequisites>`. Then run:
+Running the client in developer mode from your local tree is very similar to
+running the released versions of the client; just replace ``letsencrypt-auto``
+with ``letsencrypt-dev``:
 
 .. code-block:: shell
 
-   ./bootstrap/dev/venv.sh
+   git clone https://github.com/letsencrypt/letsencrypt
+   cd letsencrypt
+   ./letsencrypt-dev
 
-Activate the virtualenv:
 
-.. code-block:: shell
+Find issues to work on
+----------------------
 
-   source ./$VENV_NAME/bin/activate
+You can find the open issues in the `github issue tracker`_.  If you're
+starting work on something, post a comment to let others know and seek
+feedback on your plan where appropriate.
 
-This step should prepend you prompt with ``($VENV_NAME)`` and save you
```

Comment:
So much of this documentation relies on having having activated the `virtualenv`. For instance, without doing this, to run `tox` users with this setup will have to run `$XDG_DATA_HOME/letsencrypt-dev/bin/tox` (and this is if they are lucky enough to have the environment variable set). Even worse, this is not explained in the documentation below so people will follow the instructions perfectly and get `command not found: tox`.